packages/coinstac-server-core/package.json

```diff
@@ -45,7 +45,6 @@
     "jsdoc": "^3.4.0",
     "minami": "^1.1.1",
     "nyc": "^6.4.4",
-    "pouchy": "^9.0.2",
```

Comment:
This is never used by _coinstac-server-core_ as we’re relying on _coinstac-common_ for all DB goodness.